internal/webhook/external/bitbucket.go

```diff
@@ -122,15 +122,7 @@ func (b *bitbucketWebhookReceiver) getHandler(requestBody []byte) http.HandlerFu
 			return
 		}
 
-		payload := struct {
-			Repository struct {
-				Links struct {
-					HTML struct {
-						Href string `json:"href"`
-					} `json:"html"`
-				} `json:"links"`
-			} `json:"repository"`
-		}{}
+		payload := new(bitBucketPushEvent)
```

Comment:
Can't see a good reason for this to be a pointer, but apart from that, this probably causes a bit of an issue on +126 where `&payload` would now be a `**bitBucketPushEvent` instead of a `*bitBucketPushEvent`.

```suggestion
		payload := bitBucketPushEvent{}
```